_includes/head.html

```diff
@@ -1,11 +1,12 @@
 {% assign chapter = page.url | regex_extract: '^/([^/]+)/' %}
+{% assign description = page.excerpt | strip_html | strip_newlines | truncate: 160 %}
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-
-  <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
-  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
+  <title>{% if site.data.seo[chapter].title %}{{ site.data.seo[chapter].title }}{% else %}ePages{% endif %}</title>
+  <meta name="description" content="{% if page.excerpt %}{{ description }}{% else %}{{ site.data.seo[chapter].description }}{% endif %}">
```

Comment:
In the line 2, description is assigned as: page.excerpt | strip_html | strip_newlines | truncate: 160